common/conf/conf_cli_help.cpp

```diff
@@ -35,7 +35,62 @@ const size_t max_width = 78;
 const string indent = "   ";   // 3 spaces
 const string separator = "  "; // 2 spaces
 
-const Option help_option = {
+const Paths default_paths = Paths {};
+
+const vector<CliOption> common_global_options = {
+	CliOption {
+		.long_option = "config",
+		.short_option = "c",
+		.description = "Configuration FILE path",
+		.default_value = default_paths.GetConfFile(),
+		.parameter = "FILE"},
+	CliOption {
+		.long_option = "fallback-config",
+		.short_option = "b",
+		.description = "Fallback configuration FILE path",
+		.default_value = default_paths.GetFallbackConfFile(),
+		.parameter = "FILE"},
+	CliOption {
+		.long_option = "data",
+		.short_option = "d",
+		.description = "Mender state data DIRECTORY path",
+		.default_value = default_paths.GetPathDataDir(),
+		.parameter = "DIR"},
+	CliOption {
+		.long_option = "log-file",
+		.short_option = "L",
+		.description = "FILE to log to",
+		.parameter = "FILE"},
+	CliOption {
+		.long_option = "log-level",
```

Comment:
It's unfortunate that these need to be specified separately from the options that are actually used in parsing. Isn't there a way we can unify them?

Reply:
I thought about this too at the beginning.

Once I merged everything in the same namespace, I think it makes sense to do it. I'll try.